packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -139,9 +143,16 @@ class MessageGroup extends Widget<Properties> {
   _getTimeValue(timestamp: Date | string | number): string {
     const options: Intl.DateTimeFormatOptions = { hour: '2-digit', minute: '2-digit', hour12: false };
```

Comment:
Did we approve that we should use this format by default? I mean 'hour12: false' without using AM/PM. I didn't find any information in specification.

Reply:
Yeah, it looks strange at least because `hour12: false`, it will be not expected in EN locale. 
I believe it should be smth like "hh:mm" or like this (please find similar solution in DateBox).
And I believe we should always use `dateLocalization.format`, not `date.toLocaleTimeString`

Reply:
We were going to think about it on 24 July here: https://github.com/DevExpress/DevExtreme/pull/27815#discussion_r1689579976. But I don't see any cards that were created after the discussion.

Reply:
now it used `dateLocalization.format` in both cases. Default format is `HH:mm` ed2574f9d5f3c5a301c284eb8acaba16c73321ad